src/metabase/models/permissions.clj

```diff
@@ -731,10 +749,16 @@
    "Valid details perms graph for a database"))
 
 (def ^:private StrictDBPermissionsGraph
-  {su/IntGreaterThanZero {(s/optional-key :data) StrictDataPermissionsGraph
-                          (s/optional-key :download) DownloadPermissionsGraph
-                          (s/optional-key :data-model) DataModelPermissionsGraph
-                          (s/optional-key :details) DetailsPermissions}})
+  (s/conditional
+   :execute
```

Comment:
I'm confused by this and `update-global-execution-permission`, I expected `execute` to follow `details` closely. This seems to be for a global, non-db-id check, but I'm not sure why that's needed. If it's for http, I'd rather disable http actions completely for the moment until we address the datasource concept or similar.

Reply:
I did this because in the OSS version "All Users" either can execute all actions independently of DB or they can execute no actions at all. Can you plausibly do that for `details`? If yes, how?